verification/test-wdls/TestExomeGermlineSingleSample.wdl

```diff
@@ -0,0 +1,198 @@
+                            ExomeGermlineSingleSample.agg_insert_size_metrics,
+                            ExomeGermlineSingleSample.agg_pre_adapter_detail_metrics,
+                            ExomeGermlineSingleSample.agg_pre_adapter_summary_metrics,
+                            ExomeGermlineSingleSample.agg_quality_distribution_pdf,
+                            ExomeGermlineSingleSample.agg_quality_distribution_metrics,
+                            ExomeGermlineSingleSample.agg_error_summary_metrics,
+                            ExomeGermlineSingleSample.duplicate_metrics,
+                            ExomeGermlineSingleSample.gvcf_summary_metrics,
+                            ExomeGermlineSingleSample.gvcf_detail_metrics,
+                            ExomeGermlineSingleSample.hybrid_selection_metrics,
+                            ExomeGermlineSingleSample.output_cram,
+                            ExomeGermlineSingleSample.output_cram_index,
+                            ExomeGermlineSingleSample.output_cram_md5,
+                            ExomeGermlineSingleSample.validate_cram_file_report,
+                            ExomeGermlineSingleSample.output_vcf,
+                            ExomeGermlineSingleSample.output_vcf_index
+                            ], # Array[File] outputs
+                            ExomeGermlineSingleSample.quality_yield_metrics,
+                            ExomeGermlineSingleSample.unsorted_read_group_base_distribution_by_cycle_pdf,
+                            ExomeGermlineSingleSample.unsorted_read_group_base_distribution_by_cycle_metrics,
+                            ExomeGermlineSingleSample.unsorted_read_group_insert_size_histogram_pdf,
+                            ExomeGermlineSingleSample.unsorted_read_group_insert_size_metrics,
+                            ExomeGermlineSingleSample.unsorted_read_group_quality_by_cycle_pdf,
+                            ExomeGermlineSingleSample.unsorted_read_group_quality_by_cycle_metrics,
+                            ExomeGermlineSingleSample.unsorted_read_group_quality_distribution_pdf,
+                            ExomeGermlineSingleSample.unsorted_read_group_quality_distribution_metrics,
+                            # File? outputs
+                            select_all([ExomeGermlineSingleSample.cross_check_fingerprints_metrics]),
+                            select_all([ExomeGermlineSingleSample.fingerprint_summary_metrics]),
+                            select_all([ExomeGermlineSingleSample.fingerprint_detail_metrics]),
+                            select_all([ExomeGermlineSingleSample.output_bqsr_reports]),
+                            select_all([ExomeGermlineSingleSample.output_bam]),
+                            select_all([ExomeGermlineSingleSample.output_bam_index]),
+  ])
+
+  # Collect all of the pipeline metrics into a single Array[String]
+  Array[String] pipeline_metrics = flatten([
```

Comment:
Are all of pipeline_metrics in pipeline_outputs? If so, could you collect the former first and then add them to pipeline_outputs (where pipeline_outputs initially contains only non-metrics).